frappe/public/js/frappe/form/controls/select.js

```diff
@@ -150,24 +156,28 @@ frappe.ui.form.add_options = function (input, options_list, sort) {
 	};
 })(jQuery);
 
-function parse_option(v) {
+function parse_option(v, dt, df) {
```

Comment:
```suggestion
function parse_option(v, doctype, fieldname) {
```

Reply:
Done

And many thank's to take time to explain to me how use frappe standards, and finally review this PR.
In hope it will be merge by Frappe Technical Team